Gemfile

```diff
@@ -10,7 +10,7 @@ gem "rubocop", "<= 0.35.1"
 gem "simplecov", "~> 0.9"
 gem "coveralls", "~> 0.7"
 gem "yard", "~> 0.9"
-gem "yard-doctest", "= 0.1.7"
+gem "yard-doctest"
```

Comment:
The dependency was set to `~> 0.1.6` before. I think it should be set to `~> 0.1.8` now, just in case something changes when `0.2.0` comes out.